src/vivarium_gates_iv_iron/model_specifications/gates_iv_iron.yaml

```diff
@@ -20,8 +20,6 @@ components:
 configuration:
     input_data:
         input_draw_number: 0
-        location: 'South Asia'
```

Comment:
I think you can leave this so you don't need the command line arg all the time (e.g. I use south asia for all testing and just want to `simulate run /path/to/gates_iv_iron.yaml` and have it do something sensible by default.  Then CLI args can override the default in the spec.